pyomo/contrib/pynumero/src/amplmp-3.1.0.patch

```diff
@@ -2,8 +2,21 @@ diff --git a/CMakeLists.txt b/CMakeLists.txt
 index 523faa7..2523b22 100644
 --- a/CMakeLists.txt
 +++ b/CMakeLists.txt
-@@ -388,9 +388,6 @@  ** Disable AMPL testing
-   enable_cxx11(benchmark)
+@@ -336,8 +336,6 @@ check_cxx_source_compiles("
```

Comment:
My one concern about changing this patch - isn't this one of the files that we discussed on Tuesday? As in, it might become duplicated? We need to make sure _this_ version is being used universally.

Reply:
Yes: this is exactly my concern with #1904